src/Sarif.Viewer.VisualStudio/Models/FixModel.cs

```diff
@@ -1,42 +1,23 @@
 ﻿// Copyright (c) Microsoft. All rights reserved. 
 // Licensed under the MIT license. See LICENSE file in the project root for full license information. 
 
-using System;
-using System.Collections.Generic;
 using System.Collections.ObjectModel;
-using System.IO;
-using System.Linq;
-using System.Text;
-using EnvDTE;
-using EnvDTE80;
-using Microsoft.CodeAnalysis.Sarif;
-using Microsoft.VisualStudio.Shell;
 
 namespace Microsoft.Sarif.Viewer.Models
 {
     public class FixModel : NotifyPropertyChangedObject
     {
         protected string _description;
         protected ObservableCollection<ArtifactChangeModel> _artifactChanges;
-        protected DelegateCommand<FixModel> _previewFixCommand;
-        protected DelegateCommand<FixModel> _applyFixCommand;
-        protected static Dictionary<string, FixOffsetList> s_sourceFileFixLedger = null;
-        protected static string s_sourceFileFixLedgerFileName = "SourceFileChangeLedger.json";
-        protected readonly IFileSystem _fileSystem;
 
         public delegate void FixAppliedHandler();
 
-        public FixModel(string description, IFileSystem fileSystem)
```

Comment:
>FixModel [](start = 15, length = 8)

We remove all the code related to applying fixes directly to files on disk. Replaced with `FixSuggestedAction*.cs`